casper/src/test/scala/coop/rchain/casper/util/rholang/Resources.scala

```diff
@@ -1,12 +1,29 @@
 package coop.rchain.casper.util.rholang
+
+import cats.implicits._
 import cats.effect.Resource
+import cats.effect.implicits._
 import coop.rchain.catscontrib.TaskContrib._
+import coop.rchain.rholang.interpreter.Runtime
 import coop.rchain.rholang.Resources.mkRuntime
 import coop.rchain.shared.{Log, StoreType}
 import monix.eval.Task
 import monix.execution.Scheduler
 
 object Resources {
+
+  def mkRuntimeAndManager(
```

Comment:
can you parameterize this in F instead of hardcoding Task?

Reply:
nevermind this. I've just realized that mkRuntime depends on Task and the changes would go pretty deep.